src/Model/BookManager.php

```diff
@@ -99,4 +99,16 @@ public function delete($id)
         }
         return $statement->execute();
     }
+
+    public function selectByTenPerPage(int $page)
+    {
+        // prepared request
+        $bookPerPage = 10;
+        $offset = 10 * ($page - 1);
+        $statement = $this->pdo->prepare('SELECT * FROM ' . self::TABLE . ' LIMIT :limit OFFSET :offset');
+        $statement->bindValue('limit', $bookPerPage, \PDO::PARAM_INT);
+        $statement->bindValue('offset', $offset, \PDO::PARAM_INT);
+        $statement->execute();
+        return $statement->fetchAll();
```

Comment:
You should check for errors, because prepare() can return false and you can't bind if prepare() return false.